submissions/f3d0t/oop_exercise/index.js

```diff
@@ -0,0 +1,75 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/f3d0t/f3d0t/tree/main/a_tiny_js_world_post_oop
+   Web app: https://f3d0t.github.io/f3d0t/a_tiny_js_world_post_oop/
+   */
+
+class Inhabitant {
+	constructor(name, species, gender, legs, saying) {
+		this.name = name;
+		this.species = species;
+		this.gender = gender;
+		this.legs = legs;
+		this.saying = saying;
+	}
+
+	toString() {
+		const templateArray = [
+			`HI! My name is ${this.name}.`, 
+			`I belong to the specie of ${this.species}.`, 
+			`My gender is ${this.gender}.`, 
+			`Saying: "${this.saying}".`, 
+			`I have ${this.legs} legs.`
+		];
+		return templateArray.join(" ");
```

Comment:
1. Will `templateArray` contain templates after evaluation?
2. Do we need a variable here at all?

Reply:
No, It will not last after evaluation, cause this variable has only this method scope. It was introduced for readability improvement. I can get rid of it if it is redudant